types/src/proto/proof.proto

```diff
@@ -51,6 +51,21 @@ message SparseMerkleProof {
   repeated bytes non_default_siblings = 3;
 }
 
+message AccumulatorConsistencyProof {
+  // The root hashes of the frozen subtrees that form the small accumulator.
+  // Note that none of these hashes should be default hash.
+  repeated bytes frozen_subtree_roots = 1;
+
+  // The total number of siblings.
+  uint32 num_siblings = 2;
```

Comment:
Only needs to be a uint8

Reply:
I don't think there's a `uint8` type in protobuf.